<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>627</width>
    <height>410</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="1" column="0">
    <widget class="QTextEdit" name="textEdit_3">
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.AppleSystemUIFont'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Open a data file and check the header information. At this stage, you can adjust the macro time resolution (dT) and the micro time resolution (dt). &lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;The binning factor scales the micro time. Note: micro times and macro times are counted in units of dT and dt, respecivelty. Binning can improve the single-to-noise ratio, SNR, but affects the resolution.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="7" column="1">
       <widget class="QLabel" name="label_27">
        <property name="text">
         <string>dt [ps]</string>
        </property>
       </widget>
      </item>
      <item row="8" column="2" colspan="4">
       <widget class="QComboBox" name="comboBox_2">
        <item>
         <property name="text">
          <string>1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>8</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>16</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label_25">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>File format</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLabel" name="label_26">
        <property name="text">
         <string>dT [ns]</string>
        </property>
       </widget>
      </item>
      <item row="10" column="1" colspan="5">
       <widget class="QPlainTextEdit" name="plainTextEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="acceptDrops">
         <bool>false</bool>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOn</enum>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="6" column="2" colspan="4">
       <widget class="QLineEdit" name="lineEdit_14">
        <property name="acceptDrops">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>50.0</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2" colspan="4">
       <widget class="QComboBox" name="comboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <item>
         <property name="text">
          <string>PTU</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>SPC-130</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>SPC-630</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="9" column="1" colspan="5">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>TTTR Header:</string>
        </property>
       </widget>
      </item>
      <item row="7" column="2" colspan="4">
       <widget class="QLineEdit" name="lineEdit_15">
        <property name="acceptDrops">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>1.0</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QLabel" name="label_30">
        <property name="text">
         <string>Binning</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QToolButton" name="toolButton">
        <property name="text">
         <string>help</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLineEdit" name="lineEdit_16">
        <property name="dragEnabled">
         <bool>true</bool>
        </property>
        <property name="placeholderText">
         <string>Drop file here</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <action name="actionUpdate_Values">
   <property name="text">
    <string>Update Values</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>lineEdit_14</sender>
   <signal>textEdited(QString)</signal>
   <receiver>actionUpdate_Values</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>649</x>
     <y>59</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_15</sender>
   <signal>textEdited(QString)</signal>
   <receiver>actionUpdate_Values</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>649</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox_2</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>actionUpdate_Values</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>649</x>
     <y>106</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>textEdit_3</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>431</x>
     <y>14</y>
    </hint>
    <hint type="destinationlabel">
     <x>202</x>
     <y>242</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
